### PR TITLE
Adding the KWN-Deform package

### PR DESCRIPTION
**Pull request by douglowe**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by samuel-hg-engel** [comment]
Hi, just commenting here that I would be happy to be listed as one of the maintainers for this package.

**Comment by mbignon** [comment]
Hi, I am happy to belong to the maintainers list for this package

**Comment by douglowe** [comment]
I am also willing to be listed as a maintainer for this package.

**Comment by douglowe** [comment]
https://github.com/orgs/conda-forge/teams/staged-recipes This is ready for review

File: recipes/KWN-deform/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (ocefpaf) [inline recipes/KWN-deform/meta.yaml:1]: Ping!

Comment (douglowe) [reply recipes/KWN-deform/meta.yaml]: ahh, yes - all cleaned up now!

File: recipes/KWN-deform/build.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+AR="${AR:-ar}"
+
+
+export FPM_FC="${FC}"
+export FPM_AR="${AR}"
+export FPM_LDFLAGS="${LDFLAGS}"
+export FPM_FFLAGS="-fbounds-check -ffree-line-length-0 -fimplicit-none -O3 -DWITH_QP=1"
+fpm build --tests
+fpm test
+fpm install --prefix "${PREFIX}"
```
Comment (ocefpaf) [inline recipes/KWN-deform/build.sh:15]: These looks good to me but I'm not an expert on modern fpm. @isuruf I'd love if you could take a quick look here before we merge it.

Comment (douglowe) [reply recipes/KWN-deform/build.sh:15]: @ocefpaf & @isuruf - have you had chance to check these for us?

Comment (ocefpaf) [reply recipes/KWN-deform/build.sh:15]: If @isuruf can't check this today I'll merge it this afternoon and we'll see how this perform.

Comment (douglowe) [reply recipes/KWN-deform/build.sh:15]: Would it be possible to merge this in to conda forge now?

**Comment by douglowe** [comment]
@ocefpaf - is it possible to add this package to conda-forge?

**Pull request merged**